src/ServiceControlInstaller.PowerShell/Cmdlets/MonitoringInstances/NewMonitoringInstance.cs

```diff
@@ -44,7 +44,7 @@ public class NewMonitoringInstance : PSCmdlet
         public string ErrorQueue { get; set; }
 
         [Parameter(Mandatory = true, HelpMessage = "Specify the NServiceBus Transport to use")]
-        [ValidateSet("AzureServiceBus", "AzureStorageQueue", "MSMQ", "SQLServer", "RabbitMQ", "AmazonSQS")]
+        [ValidateSet(TransportNames.AzureServiceBusForwardingTopology, TransportNames.AzureServiceBusEndpointOrientedTopology, TransportNames.AzureServiceBus, TransportNames.AzureStorageQueue, TransportNames.MSMQ, TransportNames.SQLServer, TransportNames.RabbitMQDirectRoutingTopology, TransportNames.RabbitMQConventionalRoutingTopology, TransportNames.AmazonSQS)]
```

Comment:
Would it make sense to move these into a public static property called `TransportNames.All`?

Reply:
@mikeminutillo I think this would not help in this scenario. The attribute parameter has to be compile-time constant which is impossible for `string[]` type.